web-common/src/features/alerts/data-tab/AlertDialogDataTab.svelte

```diff
@@ -31,7 +31,8 @@
       label: d.label?.length ? d.label : d.expression,
     })) ?? [];
 
-  $: hasAtLeastOneFilter = $form.whereFilter.cond.exprs.length > 0;
+  $: hasAtLeastOneFilter =
+    $form.whereFilter.cond.exprs.length > 0 || $form["timeRange"];
```

Comment:
We should probably get rid of this now that the time range filter pill will always be present